pulsar-client-cpp/wireshark/CMakeLists.txt

```diff
@@ -0,0 +1,56 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+cmake_minimum_required(VERSION 3.0)
+
+find_library(PROTO_LIBRARIES NAMES libprotobuf.a)
+
+find_library(WIRESHARK_LIB wireshark)
+find_library(WIRESHARK_UTIL_LIB wsutil)
+find_path(WIRESHARK_INCLUDE_PATH wireshark/config.h)
+
+include(FindPkgConfig)
+pkg_check_modules(GLIB   glib-2.0)
+
+INCLUDE_DIRECTORIES(${WIRESHARK_INCLUDE_PATH}/wireshark
+                    ${GLIB_INCLUDE_DIRS}
```

Comment:
Change ${GLIB_INCLUDE_DIRS} to ${GLIB_INCLUDE_PATH}
AND add ${PROTOBUF_INCLUDE_DIR}